### ISSUE TITLE
neaGUI package deprecated

### ISSUE THREAD

**Issue by Ju-Rien** [lgeistlinger/EnrichmentBrowser#2]
I have been looking into the code of EnrichmentBrowser to use it at it's maximal efficiency. I discovered that `nbea()` with `method = "nea"` is not working properly, that neaGUI is removed from Bioconductor as of version 3.4 ([source](https://bioconductor.org/about/removed-packages/)), and that I cannot even install it on R 3.5.0.

Here are the input/output that made me dig deeper into neaGUI's state:
```
> res.nea.reactome <- nbea(method = "nea", se = zcz.se, gs = gs.reactome.ncbi,
+                           grn = grn.reactome, perm = 100, alpha = 0.1)
Corresponding software package not found: neaGUI
Make sure that you have it installed.
Install it now? (y/n): y
Bioconductor version 3.7 (BiocInstaller 1.30.0), ?biocLite for help
BioC_mirror: https://bioconductor.org
Using Bioconductor 3.7 (BiocInstaller 1.30.0), R 3.5.0 (2018-04-23).
Installing package(s) ‘neaGUI’
Loading required package: neaGUI
Computing NEA permutations, this may take a few minutes ...
Error in nea(ags = ags, fgs = gs, network = network, nperm = perm) : 
  could not find function "nea"
In addition: Warning messages:
1: package ‘neaGUI’ is not available (for R version 3.5.0) 
2: In library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE,  :
  there is no package called ‘neaGUI’

```

**Comment by Ju-Rien** [comment]
I suggest the removal of the nea method from `nbea()`.

**Comment by lgeistlinger** [comment]
Yes, that's a relic and is going to be removed.
Thank you for reporting this.

**Comment by lgeistlinger** [comment]
Resolved in `EnrichmentBrowser 2.10.2`
http://bioconductor.org/packages/EnrichmentBrowser
